config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -3949,3 +3949,10 @@
 # Enable OAuth support
 # DEFAULT: false
 # oauth.enabled=true
+
+# ######################################
+# SAK-29406 Allow Assignment tool to grade with two decimal points
+# ######################################
+# decimal points used to grade in new assignments (created after this code was applied), 
+# old assignments (created before this code) will continue using 1 decimal point to grade
+# assignment.grading.decimals=2
```

Comment:
Can you move this to the existing section containing assignment properties? Starts at around line 1633